recipe/meta.yaml

```diff
@@ -0,0 +1,216 @@
+        - openblas-devel                         # [mxnet_blas_impl == 'openblas']
+        - libopencv >=4.6                        # [not s390x]
+        - imagecodecs                            # [win]
+        - cudatoolkit {{ cudatoolkit_version }}  # ['gpu' in str(mxnet_variant_str)]
+        - cudnn {{ cudnn_version }}              # ['gpu' in str(mxnet_variant_str)]
+      run:
+        - _mutex_mxnet {{ mxnet_variant_version }} {{ mxnet_variant_str }}
+        - cudatoolkit {{ cudatoolkit_version }}  # ['gpu' in str(mxnet_variant_str)]
+        - cudnn {{ cudnn_version }}              # ['gpu' in str(mxnet_variant_str)]
+        - {{ pin_compatible('mkl-dnn') }}        # [unix and (mxnet_blas_impl == 'mkl')]
+        - {{ pin_compatible('intel-openmp') }}   # [mxnet_blas_impl == 'mkl']
+        - _openmp_mutex                          # [linux]
+        - opencv >=4.6                           # [not s390x]
+
+    test:
+      commands:
+        - im2rec -h   # [not (win or s390x)]
+      requires:       # [x86 and mxnet_blas_impl != 'mkl']
+        - nomkl       # [x86 and mxnet_blas_impl != 'mkl']
+
+    about:
+      home: http://mxnet.io
+      license: Apache-2
+      license_file: LICENSE
+      license_family: Apache
+      summary: MXNet is a deep learning framework designed for both efficiency and flexibility
+      description: |
+        Apache MXNet (incubating) is a deep learning framework designed for both
+        efficiency and flexibility. It allows you to mix symbolic and imperative
+        programming to maximize efficiency and productivity. At its core, MXNet
+        contains a dynamic dependency scheduler that automatically parallelizes both
+        symbolic and imperative operations on the fly. A graph optimization layer on
+        top of that makes symbolic execution fast and memory efficient. MXNet is
+        portable and lightweight, scaling effectively to multiple GPUs and multiple
+        machines. MXNet is also more than a deep learning project. It is also a
+        collection of blue prints and guidelines for building deep learning systems,
+        and interesting insights of DL systems for hackers.
+      dev_url: https://github.com/apache/incubator-mxnet
+      doc_url: https://mxnet.incubator.apache.org/
+
+
+  - name: py-mxnet
+    version: {{ mxnet_version }}
+    script: build-py-mxnet.bat  # [win]
+    script: build-py-mxnet.sh   # [not win]
+
+    build:
+      number: {{ build_number }}
+
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+      host:
+        - {{ pin_subpackage('libmxnet', exact=True) }}
+        - nomkl                                # [mxnet_blas_impl == 'openblas']
+        - cython
+        - python
+        # Numpy's C API is not used
+        - numpy >=1.16.0,<=2.0.0
+        - requests >=2.20.0,<3
+        - setuptools
+      run:
+        - {{ pin_subpackage('libmxnet', exact=True) }}
+        - numpy >=1.16.0,<=2.0.0
```

Comment:
Should be `numpy>1.16.0,<2.0.0`, see https://github.com/apache/incubator-mxnet/blob/c9818480680f84daa6e281a974ab263691302ba8/python/setup.py#L33

Reply:
yeah, see above